### ISSUE TITLE
Docker release for 1.0.0

### ISSUE THREAD

**Issue by sebastien-prudhomme** [mongo-express/mongo-express#949]
It seems no Docker release has be done for version 1.0.0. Manual operation?

Maybe add it in the CI with https://github.com/docker/build-push-action

**Comment by BlackthornYugen** [comment]
I've meant to have this done a week ago but got distracted by some stuff. If it's helpful here's a multi-arch build from my fork of our docker repo: ghcr.io/blackthornyugen/mongo-express-docker:v1.0.0 

Hopefully we can have an official build up in ghcr from
https://github.com/mongo-express/mongo-express-docker/pull/82 in the coming days.

I'm not sure if we currently have any active maintainers with credentials to push to dockerhub. Once we do we will update the GitHub action from that PR to push to both ghcr and docker.io.

I'll make sure to keep this issue up to date with our progress.

**Comment by sebastien-prudhomme** [comment]
@BlackthornYugen thanks, I was not aware that there is a repo for Docker image as there is also a Dockerfile in this repo

**Comment by radokristof** [comment]
@BlackthornYugen any news on this one? That pull request seems to be merged, but the docker hub image is not updated.

**Comment by BlackthornYugen** [comment]
> @BlackthornYugen any news on this one? That pull request seems to be merged, but the docker hub image is not updated.

Sorry, been traveling and haven't been able to finish. The last hold up was related to basic auth being required and we are worried this would break a lot of deployments.

**Comment by radokristof** [comment]
@BlackthornYugen no problem! Do you have a rough estimate for that one?

**Comment by BlackthornYugen** [comment]
> @BlackthornYugen no problem! Do you have a rough estimate for that one?

I had thought I would find time while traveling but it hasn't worked out that way so far. I wanted to do a few smoke tests before a release but my setup is broken somehow. Probably need a couple hours to sort through it. 

I'm back on October 3rd and will have the time in the first couple of days. So I hope to have it out before October 6th?

**Comment by radokristof** [comment]
@BlackthornYugen any updates on this? Thanks!

**Comment by BlackthornYugen** [comment]
I created a release candidate for 1.0.1 (1.0.1-rc) on GitHub but haven't yet tested this with docker yet. I'm still finding it difficult to find time to do this. If anyone else can test the docker build with 1.0.1-rc then we can promote it to 1.0.1 and publish to npm and dockerhub.

**Comment by shakaran** [comment]
@BlackthornYugen could we confirm and release it? I would prefer new bug reports that keep delaying the release tags

**Comment by BlackthornYugen** [comment]
@shakaran absolutely, I just need someone to smoke test it. Then I can release it.

**Comment by bwgjoseph** [comment]
Just saw this https://github.com/mongo-express/mongo-express/issues/956#issuecomment-1252039762 and seem like the same issue

**Comment by BlackthornYugen** [comment]
I'm not sure that the dockerfile or compose in that repo work correctly, we have another repo for building docker images currently. On the to-do list to simplify this.

**Comment by BlackthornYugen** [comment]
Your right. I think I haven't published the rc to npm. 🙁
